Move over the protected and encrypted source name (#20)
IronCoreLabs/try-cloaked-search: README.md

```

We are protecting the `body` and `summary` fields from the original document. These fields are no longer attached to the document;
instead, the blind index tokens for `body` and `summary` are stored in the `_icl_p_body` and `_icl_p_summary` fields, respectively.

> You will also notice an `_icl_encrypted_source` field; this contains an encrypted version of the entire document, which allows Cloaked Search
> to return the original versions of any protected fields.

Reviewer: ```suggestion
You will also notice an `_icl_encrypted_source` field; this contains an encrypted version of the fields that have been protected, which allows Cloaked Search
to return them to their original versions.
```
